vespalib/src/apps/vespa-drop-file-from-cache/drop_file_from_cache.cpp

```diff
@@ -0,0 +1,32 @@
+// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#include <cstdio>
+#include <cerrno>
+#include <cstring>
+#include <fcntl.h>
+#include <unistd.h>
+
+int main(int argc, char **argv) {
+    char errorBuf[200];
+    if (argc != 2) {
+        fprintf(stderr, "%s <filename>", argv[0]);
+        return 1;
+    }
+    const char *fileName = argv[1];
+    int fh = open(fileName, O_RDONLY);
+    if (fh != -1) {
```

Comment:
consider refactoring to guard on the same form as parameter check above (handle error in 'if' and wanted path below 'if'). this will also make it more obvious that we actually return something and the end of the main function will be more satisfying.